packages/zowe-explorer/src/webviews/src/table-view/ActionsBar.tsx

```diff
@@ -80,7 +81,8 @@ export const ActionsBar = (props: ActionsProps) => {
       </h3>
       <span style={{ display: "flex", alignItems: "center", marginBottom: "0.25em" }}>
         <p style={{ fontSize: "0.9em", paddingTop: "2px", marginRight: "0.75em" }}>
-          {props.selectionCount === 0 ? "No" : props.selectionCount} item{props.selectionCount > 1 || props.selectionCount === 0 ? "s" : ""} selected
+          {props.selectionCount === 0 ? l10n.t("No") : props.selectionCount} {l10n.t("item")}
+          {props.selectionCount > 1 || props.selectionCount === 0 ? l10n.t("s") : ""} {l10n.t("selected")}
```

Comment:
Not sure that we want to have the character `"s"` as a translation term. Would we want to do this instead?
```suggestion
          {props.selectionCount === 0 ? l10n.t("No") : props.selectionCount} 
          {props.selectionCount > 1 || props.selectionCount === 0 ? l10n.t("items") : l10n.t("item")} {l10n.t("selected")}
```